bin/runtime/src/custom_migrations.rs

```diff
@@ -0,0 +1,27 @@
+use sp_std::marker::PhantomData;
+
+use super::*;
+use frame_election_provider_support::SortedListProvider;
+use frame_support::{log, traits::OnRuntimeUpgrade, weights::Weight};
+
+pub struct UpgradeToBags<T>(PhantomData<T>);
```

Comment:
I need to test it, but this migration will be triggered at every runtime update, right? Unless we remove this code from the runtime after this update :)

Anyway, this requires a bit of thought whether we can somehow make it better.

Reply:
To run it conditionally we would need some methods as for storage version eg current/next storage version but I couldn't find anything like that. So I would just delete the mod and remove the upgrade from Executive leaving a code artifact documenting the upgrade.